lib/active_model/serializer/adapter/json_api/link.rb

```diff
@@ -0,0 +1,34 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        class Link
```

Comment:
I think so maybe, but that's what the ApiObjects namespace was for, and also the two RFCs